cmd/commands/init.go

```diff
@@ -0,0 +1,122 @@
+package commands
+
+import (
+	"io/ioutil"
+	"path"
+
+	"github.com/urfave/cli"
+
+	cmn "github.com/tendermint/go-common"
+)
+
+var InitCmd = cli.Command{
+	Name:      "init",
+	Usage:     "Initialize a basecoin blockchain",
+	ArgsUsage: "",
+	Action: func(c *cli.Context) error {
+		return cmdInit(c)
+	},
+	Flags: []cli.Flag{
+		ChainIDFlag,
+	},
+}
+
+func cmdInit(c *cli.Context) error {
```

Comment:
Is there a functional difference (besides the directory) between `basecoin init` and `tendermint init` noticed code is totally different, here if just looks like we're writing in some files, don't fully understand everything happening in `tendermint init`. If there is no other functional difference we should find a way to rectify the two functions to be able to call some core common code if possible. The refactor to cobra should make this easier.